src/rpc/api/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from "./interface";
-export * from "./mock";
-export * from "./api";
+export * from "./interfaces";
+export * from "./mock/validator";
+export * from "./modules";
```

Comment:
I don't think we should keep mock implementations in src directory. I would recommend to place mock implementation in test directory in utils/helpers directory. It confuses because it can lead people to think it's not implemented, it messes up code coverage and it's only used in tests (aka you don't won't to bundle it in release version). Would love to hear you opinion on this :)

Reply:
That sounds reasonable to me. Its only useful for testing purposes.